extensionlauncher/extensionlauncher_windows_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
We will be responsible for testing. Do you expect that we will run any tests from Windows? If not this is code we do not need/want.